Gears/ScriptHUD.cpp

```diff
@@ -539,14 +547,16 @@ void drawVehicleWheelInfo() {
                 //fmt::format("Health: \t{:.3f}", wheelsHealt[i]),
                 fmt::format("Power: \t{:.3f}", wheelsPower[i]),
                 fmt::format("Brake: \t{:.3f}", wheelsBrake[i]),
-                fmt::format("{}ABS~w~ | {}TSC~w~ | {}ESP{}", 
+                fmt::format("{}ABS~w~ {}TCS~w~ {}GRD~w~ {}ESC{}",
                     g_vehData.mWheelsAbs[i] ? "~r~" : "",
                     g_vehData.mWheelsTcs[i] ? "~r~" : "",
+                    g_vehData.mWheelsOnGround[i] ? "" : "~r~",
```

Comment:
I intended this last line to just be assists. Wheels-on-ground is already covered with the background color.